### PR TITLE
#85946 Added list of tokenizers don't support keep types token filter

### PR DESCRIPTION
**Pull request by amitmbm**
fix for #85946

### CODE REVIEW

**Comment by amitmbm** [comment]
@javanna @romseygeek please review and let me know if you have any feedbacks.
Thanks in advance.

File: docs/reference/analysis/tokenfilters/keep-types-tokenfilter.asciidoc [hunk 1]
```diff
@@ -20,9 +20,12 @@ produce a variety of token types, including `<ALPHANUM>`, `<HANGUL>`, and
 <<analysis-lowercase-tokenizer,`lowercase`>> tokenizer, only produce the `word`
 token type.
 
-Certain token filters can also add token types. For example, the 
+Certain token filters can also add token types. For example, the
 <<analysis-synonym-tokenfilter,`synonym`>> filter can add the `<SYNONYM>` token
 type.
+
+Some tokenizers don't support this token filter, for example keyword, simple_pattern, and
+simple_pattern_split tokenizers.
```
Comment (javanna) [inline docs/reference/analysis/tokenfilters/keep-types-tokenfilter.asciidoc:28]: I would add something like "as they don't support setting the token type attribute."

Comment (amitmbm) [reply docs/reference/analysis/tokenfilters/keep-types-tokenfilter.asciidoc:28]: Thanks @javanna so entire sentence would be like 
```
Some tokenizers don't support this token filter, for example keyword, simple_pattern, and
simple_pattern_split tokenizers, as they don't support setting the token type attribute.
```
let me know if this sounds OK to you and I would make the required change.

**Review by javanna — APPROVED** [review]
thanks @amitmbm

**Pull request merged**